content/2.components/aurora-background.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Aurora Background
+description: A subtle Aurora or Southern Lights background for your website.
+navBadges:
+  - value: New
+    type: lime
+---
+
+::ComponentLoader{label="Preview" componentName="AuroraBackgroundDemo" type="examples" id="aurora-background"}
+::
+
+## API
+
+| Prop Name        | Type      | Default | Description                                                               |
+| ---------------- | --------- | ------- | ------------------------------------------------------------------------- |
+| `class`          | `string`  | `-`     | Additional CSS classes to apply to the component for styling.             |
+| `radialGradient` | `boolean` | `true`  | Determines whether a radial gradient effect is applied to the background. |
+
+You can copy and paste the `tailwind.config.ts` code.
+
+```ts{1-3, 13, 16-23, 27, 31-40} [tailwind.config.ts]
```

Comment:
This is not required as we are limiting this code to component only.

Reply:
Done